src/App.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    return title.toLowerCase().trim().includes(query.toLowerCase())
```

Comment:
split it with variables 
also, create a separate function for check `str` includes `substr`
also, you can move normalized these variables into this function